examples/themes/my-partial-theme/data/index.js

```diff
@@ -20,11 +20,18 @@ module.exports = ({
   // The Introspection Query Response after all the augmentation and metadata directives
   // have been applied to it
   introspectionResponse,
+  // All the options that are specifically for the introspection related behaviors, such a this area.
+  introspectionOptions,
   // A GraphQLSchema instance that was constructed from the provided introspectionResponse
   graphQLSchema: _graphQLSchema,
+  // All of the SpectaQL options in case you need them for something.
+  allOptions: _allOptions,
 }) => {
   const introspectionManipulator = new IntrospectionManipulator(
-    introspectionResponse
```

Comment:
NOT passing the same options that we passed to the microfiber instance in the `augmenters.js` was effectively a bug 🐛 since the default `removeUnusedTypes` was `true`, but in SpectaQL we really wanted it to be `false`...but it was in fact acting as `true` thanks to here no matter what we thought we'd done.